src/journal/journald-config.c

```diff
@@ -401,6 +402,16 @@ static const char* const split_mode_table[_SPLIT_MAX] = {
 DEFINE_STRING_TABLE_LOOKUP(split_mode, SplitMode);
 DEFINE_CONFIG_PARSE_ENUM(config_parse_split_mode, split_mode, SplitMode);
 
+static const char* const audit_set_mode_table[_AUDIT_SET_MODE_MAX] = {
+        [AUDIT_DISABLE] = "no",
+        [AUDIT_ENABLE]  = "yes",
```

Comment:
I'd strongly prefer if the internal enum names and their string representations would match, i.e. please rename AUDIT_DISABLE to AUDIT_NO, and similar for AUDIT_ENABLE

Reply:
As I wrote in the above, I intentionally named so, but ok, will change the name, and add comment at the definition.